indy_common/authorize/authorizer.py

```diff
@@ -192,3 +197,72 @@ def authorize(self,
                   auth_constraint: AuthConstraintForbidden,
                   auth_action: AbstractAuthAction):
         return False, str(auth_constraint)
+
+
+class EndorserAuthorizer(AbstractAuthorizer):
+    NO_NEED_FOR_ENDORSER_ROLES = {ENDORSER, TRUSTEE, STEWARD}
+    NO_NEED_FOR_ENDORSER_ROLES_STR = [ENDORSER_STRING, TRUSTEE_STRING, STEWARD_STRING]
```

Comment:
Any reason why this is a list instead of set?

Reply:
It's used in an error message only, and the order is important there while there is no checks for membership.